tests/suites/appdata/appdata.sh

```diff
@@ -0,0 +1,54 @@
+test_appdata_int() {
+  if [ "$(skip 'test_appdata_int')" ]; then
+    echo "==> TEST SKIPPED: appdata int tests"
+    return
+  fi
+
+  (
+    set_verbosity
+
+    cd .. || exit
+
+    run "run_appdata_basic"
+  )
+}
+
+run_appdata_basic() {
+    echo
+
+    file="${TEST_DIR}/appdata-basic.txt"
+
+    ensure "appdata-basic" "${file}"
+
+    juju deploy ./acceptancetests/repository/charms/appdata-source
+    juju deploy -n 2 ./acceptancetests/repository/charms/appdata-sink
```

Comment:
Question, should we link to the acceptance tests here, previously I've just re-included them next to the test. Then each test manages what they deploy, rather than a global repo of charms/bundles. The advantage is that it's easier to see whom owns what etc?

Reply:
Yeah, that's a good call - in that case there's no old test that needs them, I'll just move them here.